modules/nested-tabs/widgets/nested-tabs.php

```diff
@@ -416,7 +416,7 @@ protected function register_controls() {
 				'name' => 'tabs_title_background_color',
 				'types' => [ 'classic', 'gradient' ],
 				'exclude' => [ 'image' ],
-				'selector' => "{$nested_tabs_heading_selector_class} > .e-n-tab-title:not( .e-active ):not( :hover )",
+				'selector' => "{$nested_tabs_heading_selector_class} > .e-n-tab-title:not( .e-active )",
```

Comment:
Does this work correctly for both mobile and desktop? I assume we do need the `:not( :hover )` for the desktop version.

Reply:
Correct, 
The hover worked but something else broke.
I fixed it now.